src/Logging.jl

```diff
@@ -47,13 +92,17 @@ for (fn,lvl,clr) in ((:debug,    DEBUG,    :cyan),
                      (:critical, CRITICAL, :red))
 
     @eval function $fn(logger::Logger, msg...)
-        if $lvl >= logger.level
-            logstring = string(Libc.strftime("%d-%b %H:%M:%S",time()),":",$lvl, ":",logger.name,":", msg...,"\n")
-            if isa(logger.output, Base.TTY)
-                Base.print_with_color($(Expr(:quote, clr)), logger.output, logstring )
-            else
-                print(logger.output, logstring)
-                flush(logger.output)
+        if $lvl <= logger.level
+            for output in logger.output
+                logstring = string(strftime("%d-%b %H:%M:%S",time()),":",$lvl, ":",logger.name,":", msg...,"\n")
+                if isa(output, Base.TTY)
```

Comment:
Could you use pattern matching on `log()` here instead of the `if elseif else` statement? One call to `log()` which specializes for output of type `Base.TTY`, `IO`, and `Syslog`?